superset/models/helpers.py

```diff
@@ -1314,7 +1314,7 @@ def dttm_sql_literal(self, dttm: sa.DateTime, col_type: Optional[str]) -> str:
         if sql:
             return sql
 
-        return f'{dttm.strftime("%Y-%m-%d %H:%M:%S.%f")}'
+        return f"""'{dttm.strftime("%Y-%m-%d %H:%M:%S.%f")}'"""
```

Comment:
This is the wrong place to fix this... the single quotes are a database-specific construction, and adding them should be delegated to the database.

Where is `dttm_sql_literal` called? Why are other DBs quoting this correctly but not Presto?

Reply:
So i was following a pattern i saw in the original `dttm_sql_literal`, right now the original is now an override for `SqlaTable` queries vs. `Query` queries

https://github.com/apache/superset/blob/976e33330fa5e0ae4655a4011cee2eb642e12b1b/superset/connectors/sqla/models.py#L382